ulist/python/ulist/core.py

```diff
@@ -446,6 +446,11 @@ def starts_with(self, elem: str) -> UltraFastList:
         assert isinstance(self._values, StringList)
         return UltraFastList(self._values.starts_with(elem))
 
+    def str_len(self) -> "UltraFastList":
+        """Return a list on the length of strings"""
```

Comment:
Maybe """Return each element's string length of self""" would be better:)

Reply:
^_^ OK, I'll modify it latter. I am not at expressing something in English and I am trying to practice. Thanks, have a good day~